### ISSUE TITLE
2.1.14 - Unknown column 'brewerProAm' in 'field list'

### ISSUE THREAD

**Issue by codenbrew** [geoffhumphrey/brewcompetitiononlineentry#1012]
Version:
2.1.14

Installation URL:
https://tvh.brewcomp.com/

Is your installation hosted on brewcompetition.com or brewcomp.com?
brewcomp.com

Description of Issue:
This was a working installation that recently hosted a competition.  We are gearing up to host a new competition and cleared the participants, etc, by going to Admin Dashboard->Data Management->Manage then Option 1.  Unsure if this is what triggered the issue.

Now, when adding/editing a participant, the error "Unknown column 'brewerProAm' in 'field list'" is shown, which looks like a column is missing in the database? 

Related is that when adding a new participant, the error occurs but the user is added and is not visible anywhere in the system.  The system does know the participant exists, however, as it will not allow a duplicate creation.

The effect is that participants can no longer be added or edited.

For enhancements, please prepend "Enhancement - " to the title. For issues/bugs, please prepend the BCOE&M version number the title.

**Comment by geoffhumphrey** [comment]
I'm not entirely sure why the brewerProAm column wasn't added to your installation (actually, it wasn't added - the name of an unused column was changed to it). I'll investigate that. In the meantime, I've gone ahead and renamed the column manually in your installation's DB. I tested and was able to add a user. Hopefully it's smooth sailing from here on out.

For posterity, in the brewer table, the brewerJudgeBOS column was changed to brewerProAm in 2.1.4. I ran the following SQL command to change the column name and type (change prefix_ to your DB instance's prefix or delete if no prefix is used):
```
ALTER TABLE `prefix_brewer` CHANGE `brewerJudgeBOS` `brewerProAm` TINYINT(2) NULL DEFAULT NULL;
```
